README.md

```diff
@@ -123,16 +123,16 @@ see specs in *spec/integration* folder:
 
 ```ruby
 let(:stubbed_env) { Rspec::Shell::Expectations::StubbedEnv.new }
-stubbed_env.execute('./spec/scripts/function_library.sh')
+stubbed_env.execute('./path/to/script.sh')
```

Comment:
I did this in response to your other comment about the script not doing anything. Now it just uses a generic path/script to show how to call it, instead of a real, existing script